### PR TITLE
external/internal abl export TM-2785

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-Do:
- [x] `Export` turns to `Export External View` on Toggle (vice versa)
- [x] As an Internal CDA user, clicking `Export` returns the Internal View .csv (the entire ABL table, nothing is removed)
- [x] As an Internal CDA user, clicking `Export External View` returns the External View .csv (removes Status, Step Letters, Comments)
- [x] As an External CDA user, clicking `Export` returns the External View .csv

### CODE REVIEW

File: src/sass/_availableBidders.scss [hunk 1]
```diff
@@ -246,7 +246,7 @@
   }
 }
 
-.bureau-view-toggle {
+.external-internal-view-toggle {
```
Comment (elizabeth-jimenez) [inline src/sass/_availableBidders.scss:249]: should this just be 'external-view-toggle'?

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 2]
```diff
@@ -20,11 +20,11 @@ const useStepLetter = () => checkFlag('flags.step_letters');
 
 const AvailableBidderTable = props => {
   const { isCDO, isAO, isPost } = props;
-  const isCDOorAO = (isCDO || isAO);
+  const isInternalCDA = (isCDO || isAO);
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:23]: is there a benefit to setting this here as opposed to closer to where `isCDO` and `isAO` get set(before the prop drilling)? Like AvailableBidderContainer or even BureauPage, for example?

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 3]
```diff
@@ -20,11 +20,11 @@ const useStepLetter = () => checkFlag('flags.step_letters');
...
 
   // Local state
-  // Toggle view state within CDO version
-  const [cdoView, setCdoView] = useState(true);
+  // Toggle view state within Internal CDA version
+  const [internalCDAView, setInternalCDAView] = useState(true);
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:27]: can we call this `internalViewToggle` instead?

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 4]
```diff
@@ -145,6 +145,7 @@ const AvailableBidderTable = props => {
               onClick={exportBidders}
               isLoading={exportIsLoading}
               disabled={!bidders.length}
+              text={internalCDAView ? 'Export' : 'Export External View'}
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:148]: 🙌

**Comment by elizabeth-jimenez** [comment]
image
External CDA Share

File: src/actions/availableBidders.js [hunk 5]
```diff
@@ -232,9 +232,9 @@ export function availableBidderEditData(id, data, sortType = 'Name') {
   };
 }
 
-export function availableBidderExport(cdo, isSort = 'Name') {
+export function availableBidderExport(isInternalCDAExport, isSort = 'Name') {
```
[the remark this answers is not in the document]
Comment (SidelineCory24) [reply src/actions/availableBidders.js:235]: Updating to `isInternalCDAView`. `isInternalCDA === false` would mean it's an External CDA, thus an External Export, but Internal CDAs can still do the External Export.

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx (hunk 2 above)
Comment (SidelineCory24) [reply src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:23]: I look into it

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx (hunk 3 above)
Comment (SidelineCory24) [reply src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:27]: done

File: src/actions/availableBidders.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (SidelineCory24) [reply src/actions/availableBidders.js:235]: done

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 6]
```diff
@@ -114,7 +114,7 @@ const AvailableBidderTable = props => {
   const exportBidders = () => {
     if (!isLoading) {
       setExportIsLoading(true);
-      availableBidderExport(isCDOorAO, sort)
+      availableBidderExport(!internalCDAView && isInternalCDA ? false : isInternalCDA, sort)
```
[the remark this answers is not in the document]
Comment (SidelineCory24) [reply src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx]: updates made to logic, discussed offline

**Comment by elizabeth-jimenez** [comment]
> > image External CDA Share
> 
> I was leaving it as `External` because we did a 1-1 swap for `Bureau` and `External`. It also would need to go on two lines, which would end the label one-liner consistency and we're also showing "External CDA" at the top for the note, so they will have some context.
> 
> image

ok, valid points, you win 😭

File: src/Components/AvailableBidder/AvailableBidderContainer/AvailableBidderContainer.jsx [hunk 7]
```diff
@@ -4,26 +4,29 @@ import AvailableBidderTable from '../AvailableBidderTable';
...
+const AvailableBidderContainer = ({ isCDO, isAO, isPost }) => {
+  const isInternalCDA = (isCDO || isAO);
+  return (
+    <div className="position-manager-details bidder-manager-page">
+      <div className={'usa-grid-full profile-content-inner-container'}>
+        <div className="usa-grid-full">
+          <ProfileSectionTitle title="Available Bidders" icon="users" />
+        </div>
+        {
+          (isInternalCDA) &&
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderContainer/AvailableBidderContainer.jsx:16]: no need for parenthesis

**Review by elizabeth-jimenez — APPROVED** [review]
Woot! 🦉

**Pull request merged**